src/components/Button.js

```diff
@@ -46,26 +58,45 @@ export const Button = ({ disabled, invert, notFullWidth, onPress, title }) => {
       titleStyle={[
         styles.titleStyle,
         invert && styles.titleStyleInvert,
-        needLandscapeStyle && styles.titleStyleLandscape
+        needLandscapeStyle && styles.titleStyleLandscape,
+        extraLarge && styles.extraLargeTitle,
+        large && styles.largeTitle,
+        medium && styles.mediumTitle,
+        small && styles.smallTitle
       ]}
       disabledStyle={styles.buttonStyleDisabled}
       disabledTitleStyle={styles.titleStyle}
       buttonStyle={[
         styles.buttonStyle,
         invert && styles.buttonStyleInvert,
-        isDelete && styles.rejectStyle
+        isDelete && styles.rejectStyle,
+        extraLarge && styles.extraLarge,
+        large && styles.large,
+        medium && styles.medium,
+        small && styles.small
       ]}
       containerStyle={[styles.containerStyle, needLandscapeStyle && styles.containerStyleLandscape]}
       ViewComponent={invert || isDelete || disabled ? undefined : DiagonalGradient}
       useForeground={!invert}
       accessibilityLabel={`${title} ${consts.a11yLabel.button}`}
       disabled={disabled}
+      icon={{
```

Comment:
let's use our `<Icon....` passed here as `icon` instead of the possible sets from react-native-elements to be able to use other icons as well

Reply:
I updated the icon prop as a component

https://github.com/smart-village-solutions/smart-village-app-app/commit/5933f8013e0cfaaab2a97c661ecfa3b13eda8857